backend/src/dbfuncs/flashcards.py

```diff
@@ -0,0 +1,57 @@
+from dotenv import load_dotenv
+import os
+from src.models.Flashcard import Flashcard
+
+
+# Get Secret Items
+load_dotenv()
+connection_string = os.getenv("ConnectionString")
+
+# MongoDB Driver
+import motor.motor_asyncio
+client = motor.motor_asyncio.AsyncIOMotorClient(connection_string)
+
+# Create Database & Collection
+database = client.SAIO
+collection = database.flashcard
+
+# Function to create a new flashcard
+async def create_flashcard(flashcard: Flashcard):
+    document = flashcard.dict()
+    result = await collection.insert_one(document)
+    created_flashcard = await collection.find_one({"_id": result.inserted_id})
+    return Flashcard(**created_flashcard)
+
+# Read All Flashcards
+async def read_all_flashcards(category: str, user_email: str):
+    flashcards = []
+    cursor = collection.find({"category": category, "user_email": user_email})
+    async for document in cursor:
+        flashcards.append(Flashcard(**document))
+    return flashcards
+
+async def read_all_flashcards_questions(category: str, user_email: str):
+    questions = []
+    cursor = collection.find({"category": category, "user_email": user_email})
+    async for document in cursor:
+        flashcard = Flashcard(**document)
+        questions.append(flashcard.question)
+    return questions
+
+# Delete Flashcard
+async def delete_flashcard(flashcard_id: str, user_email: str):
+    result = await collection.delete_one({"question": flashcard_id, "user_email": user_email})
```

Comment:
Should this function delete by id or question? It's accepting the "flashcard_id" but deleting by "question". I think you should **delete by ID**; multiple flashcards could have the same question in different categories, meaning more than one can get deleted on accident